### ISSUE TITLE
Moodle 4.4.1 and 4.5dev get some debug messages

### ISSUE THREAD

**Issue by ralf-krause** [mebis-lp/mod_learningmap#104]
Dear developers,
the learningmap tool is very important tool for me and my teacher workshops.

Since I updated my test page to Moodle 4.5dev (20240719) I could not register the site at Moodle annymore. See https://tracker.moodle.org/browse/MDL-82556. Therefore I switched on the debugging to developer mode and I get the following debug messages for the plugin mod_learningmap. Yes, I controlled this also in my productive sites running Moodle 4.4.1+ (20240719).

I hope you will get a fix for this problem very soon.
Best regards,
Ralf

```
Callback before_standard_html_head in block_gps component should be migrated to new hook callback for core\hook\output\before_standard_head_html_generation
line 7475 of /lib/moodlelib.php: call to debugging()
line 7539 of /lib/moodlelib.php: call to {closure}()
line 73 of /lib/classes/hook/output/before_standard_head_html_generation.php: call to get_plugins_with_function()
line 718 of /lib/outputrenderers.php: call to core\hook\output\before_standard_head_html_generation->process_legacy_callbacks()
line 219 of /lib/mustache/src/Mustache/Context.php: call to core_renderer->standard_head_html()
line 138 of /lib/mustache/src/Mustache/Context.php: call to Mustache_Context->findVariableInStack()
line 31 of /var/www/vhosts/elisl.org/httpdocs/moodledata/localcache/mustache/1721551396/boost_union/__Mustache_73560842f9d6c5bc337558df4f13b989.php: call to Mustache_Context->findDot()
line 13 of /var/www/vhosts/elisl.org/httpdocs/moodledata/localcache/mustache/1721551396/boost_union/__Mustache_c4e295fe06d3b1cfa93c557453b9e4d0.php: call to __Mustache_73560842f9d6c5bc337558df4f13b989->renderInternal()
line 67 of /lib/mustache/src/Mustache/Template.php: call to __Mustache_c4e295fe06d3b1cfa93c557453b9e4d0->renderInternal()
line 216 of /lib/outputrenderers.php: call to Mustache_Template->render()
line 219 of /theme/boost_union/layout/drawers.php: call to renderer_base->render_from_template()
line 1483 of /lib/outputrenderers.php: call to include()
line 1408 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()
line 134 of /admin/settings.php: call to core_renderer->header()
```

**Comment by sh-csg** [comment]
Hi Ralf,

this is already fixed by #98 (contained in current master, but not released yet).

Best
Stefan

**Comment by ralf-krause** [comment]
Hi Stefan,
I just installed the current master on my test platform and the debug message is not shown anymore. 
Thanks!
Ralf
